soccer/planning/CompositePath.cpp

```diff
@@ -93,11 +93,18 @@ void CompositePath::draw(SystemState* const state, const QColor& color,
 
 float CompositePath::getDuration() const { return duration; }
 
-boost::optional<MotionInstant> CompositePath::destination() const {
+MotionInstant CompositePath::start() const {
     if (paths.empty()) {
-        return boost::none;
```

Comment:
Infinite paths make things kinda weird... unless you can see a good use case for them, I'd say we restrict them to be finite.